src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -24,6 +24,22 @@
     @Override
     protected void init() throws Exception {
         loadIndex(Index.ONLINE);
+        loadIndex(Index.PEOPLE);
+    }
+
+    public void testPercentilesQuery() {
+        JSONObject response = executeJdbcRequest(
+                "SELECT percentiles(age, 25.0, 50.0, 75.0, 99.9) age_percentiles" +
```

Comment:
IIRC, the non-sql percentiles aggregation query honors the order specified in the query (or it always sorts in ascending). The example response you put in the PR description does none of those. I know this is a useful fix even without that, but ideally the sql plugin should be consistent with DSL if possible.

Reply:
Unfortunately our serialization library (org.json) doesn't work well in that matter, and doesn't respect the order at all. Even if you will put percentiles into TreeMap (I've tried that in the beginning), it will still serialize it in random order, because underlying collection is hashmap. We need to move to either Jackson or GSON for that to be fixed

Reply:
Could you please double check if it's the same for plain json response (without 'format' parameter in the request url)? I suspect this was working as expected for non-jdbc queries. I just wonder if we use two separate serialization libraries for JDBC vs non-JDBC cases.

Reply:
Default works because we don't do any post-processing or deserialization there - it's just elastic output, that makes things right.

Reply:
I see, thanks for the clarification.